src/runtime_src/core/tools/common/XBUtilities.cpp

```diff
@@ -752,15 +752,16 @@ fill_xrt_versions(const boost::property_tree::ptree& pt_xrt,
     const boost::property_tree::ptree& driver = drv.second;
     std::string drv_name = driver.get<std::string>("name", "N/A");
     std::string drv_hash = driver.get<std::string>("hash", "N/A");
+    if ((boost::iequals(drv_name, "xclmgmt") || boost::iequals(drv_name, "xocl")) && (boost::iequals(driver.get<std::string>("version", "N/A"), "unknown"))) {
```

Comment:
Do you still need this? Given that if ver is unknown, you are not even populating i tin system_linux.cpp

Reply:
True, this snippet is being over-cautious. I can remove it, thanks!